### DISCUSSION TITLE
Autofocus javascript method

### DISCUSSION THREAD

**Discussion by pwbzh** [orchidjs/tom-select#416] [Ideas]
Hello,

I'm looking for an autofocus javascript method. I would like to be able to open a select via javascript and place the cursor to avoid the user having to click on.

Thanks a lot

**Comment by nikhilstacks** [comment]
you can use autofocus DOM property after accessing the DOM element.

var x = document.getElementById("myPassword").autofocus;

after (.) autofocus property will simply put focus of your cursor to that element.
